src/app/client/src/app/modules/cbse-program/components/ckeditor-tool/ckeditor-tool.component.ts

```diff
@@ -569,16 +570,29 @@ getAllVideos(offset, query) {
           const errInfo = { errorMsg: 'Video upload failed' };
           return throwError(this.cbseService.apiErrorHandling(err, errInfo));
         })).subscribe((response) => {
-          this.selectedVideo = response.result;
-          this.addVideoInEditor();
-          this.showVideoPicker = false;
-          this.showVideoUploadModal = false;
+          //Read upload video data
+          this.getUploadVide(response.result.node_id);
```

Comment:
@JagadishPujari Correct the spelling.
cc: @vinukumar-vs

Reply:
Corrected spelling